.github/workflows/macos.yml

```
@@ -233,6 +233,12 @@ jobs:
           echo '::group::raw'; cat bld/lib/curl_config.h || true; echo '::endgroup::'
           cat bld/lib/curl_config.h | grep -F '#define' | sort || true
 
+      - name: 'build-cert'
+        if: contains(matrix.name, 'SecureTransport')
```

```suggestion
        if: contains(matrix.configure, '--with-secure-transport')
```
sync condition with other step.